atomic_reactor/plugins/post_tag_by_labels.py

```diff
@@ -66,6 +66,12 @@ def get_label(label_name):
         version = get_label("version")
         release = get_label("release")
 
+        # docker tags cannot be longer than 128 chars
+        # see https://github.com/docker/docker/pull/8447
+        if len(release) > 128:
+            self.log.warn('Release is too long, docker only supports 128 char tags')
+            release = release[:127]
```

Comment:
Actually the longest tag would be `"%s-%s" % (version, release)"`, so maybe that's what we should be checking. And thinking about it more, maybe we should just fail if we get here. I can't think of any circumstance where truncating would result in something the user wanted.